test1/task1/Task1/Vector.cs

```diff
@@ -0,0 +1,131 @@
+﻿namespace Vector;
+
+using Operations;
+
+public class Vector
+{
+    private List<Element> elements;
+    private int length;
+
+    public Vector(IList<int> input)
+    {
+        this.elements = new List<Element>();
+        for (int i = 0; i < input.Count; ++i)
+        {
+            if (input[i] != 0)
+            {
+                this.elements.Add(new Element(input[i], i));
+            }
+        }
+
+        this.length = input.Count;
+    }
+
+    public bool IsNull {
+        get
+        {
+            return this.elements.Count == 0;
+        }
+    }
```

Comment:
```suggestion
    public bool IsNull => this.elements.Count == 0;
```
Это то же самое, но в шесть раз короче